### ISSUE TITLE
Ability to get postgres backend pid

### ISSUE THREAD

**Issue by oleg-slapdash** [brianc/node-postgres#2665]
I am aware that there is a SQL function get postgres backend pid. However, I am looking into a solution without an extra round-trip to the database. 

libpq has a built-in function PQbackendPID to get process pid. I assume this should be possible with JS implementation?

**Comment by sehrope** [comment]
Yes the backend pid is stored on the `pg.Client` object in the field `processID`: https://github.com/brianc/node-postgres/blob/97eea2d7a4453645e44129378215f88dff371a08/packages/pg/lib/client.js#L267

**Comment by oleg-slapdash** [comment]
My bad, the type definition `@types/pg` does not contain it and it is not specified in the documentation: https://node-postgres.com/api/client
